RZAssemblage/RZArrayAssemblage.m

```diff
@@ -77,8 +77,9 @@ - (void)removeObjectFromChildrenAtIndex:(NSUInteger)index
 {
     RZAssemblageLog(@"%p:Remove %@ at %zd", self, [self objectInChildrenAtIndex:index],  index);
     [self openBatchUpdate];
+    id object = [self.childrenStorage objectAtIndex:index];
```

Comment:
Yea, brilliant idea, forgot about those!
Implementing the integer one to directly access, and the object one to access via index path will make the code read very nicely